charts/apisix/Chart.yaml

```
@@ -31,12 +31,12 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.9.4
+version: 0.9.5
```

```suggestion
version: 0.10.0
```

In this project, when APISIX releases a new version, we will directly update its minor version, let set it to `0.10.0`